ui/app/utils/validators.js

```diff
@@ -25,17 +31,35 @@ export const number = (value, { nullable = false } = {}) => {
   return !isNaN(value);
 };
 
-/*
-the following validations return false (invalid) if the condition is met
-*/
 export const containsWhiteSpace = (value) => {
-  const validation = new RegExp('\\s', 'g'); // search for whitespace
-  return !validation.test(value);
+  return !hasWhitespace(value);
 };
 
 export const endsInSlash = (value) => {
   const validation = new RegExp('/$');
   return !validation.test(value);
 };
 
-export default { presence, length, number, containsWhiteSpace, endsInSlash };
+/*
+* General Validators
+these utils return true or false relative to the function name
+*/
+
+export const hasWhitespace = (value) => {
```

Comment:
I can put these somewhere else, they just seemed to make sense here? But if this is confusing happy to move!